code/headers/display_screen.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <cstdint>
+
+namespace r2d2::display {
+    struct st7735_128x160_s {
```

Comment:
why are `st7735_128x160_s`, `st7735_80x160_s` and `ssd1306_128x64_s` not constexpr static instances of a general `r2d2::display::DisplayScreen_s` struct

Reply:
I will merge this for now and add this item to the backlog